### PR TITLE
Wallet/Profile/User test

### PR DESCRIPTION
**Pull request by NorthDragons**
Written tests for wallet/profile/user services, implementation correction in view of identified shortcomings

### CODE REVIEW

File: finance_manager/src/main/java/com/manager/finance/security/config/SecurityConfig.java [hunk 1]
```diff
@@ -45,13 +45,13 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
...
     @Bean
     public CorsConfigurationSource corsConfigurationSource() {
+
         final CorsConfiguration configuration = new CorsConfiguration();
 
         configuration.setAllowedOrigins(Arrays.asList("*"));
         configuration.setAllowedMethods(Arrays.asList("*"));
         configuration.setAllowCredentials(false);
         configuration.setAllowedHeaders(Arrays.asList("*"));
-        configuration.setExposedHeaders(Arrays.asList("*"));
```
Comment (AlexPravda1) [inline finance_manager/src/main/java/com/manager/finance/security/config/SecurityConfig.java:54]: this line fix Headers visibility for FrontEnd, please leave it

File: finance_manager/src/main/java/com/manager/finance/user/controller/UserController.java [hunk 2]
```diff
@@ -45,6 +45,7 @@ public ResponseEntity<Void> login(@Valid UserLoginDto userLoginDto) {
 
     @Override
     public ResponseEntity<Void> logout(String authorization) {
-        return new ResponseEntity<>(HttpStatus.OK);
+        return null;
```
Comment (AlexPravda1) [inline finance_manager/src/main/java/com/manager/finance/user/controller/UserController.java:48]: we dont return even status "OK" after logout?

File: finance_manager/src/main/java/com/manager/finance/user/service/ProfileServiceImpl.java [hunk 3]
```diff
@@ -16,33 +17,36 @@
...
+    private final UserService userService;
     private final ProfileMapper profileMapper;
 
     @Override
-    public void deleteProfile(User id) {
-        Profile activeProfile = findByUserIdWithValidation(id);
+    public void deleteProfile() {
+        Profile activeProfile = findByUserIdWithValidation();
         activeProfile.setIsDeleted(true);
         profileDao.save(activeProfile);
```
Comment (AlexPravda1) [inline finance_manager/src/main/java/com/manager/finance/user/service/ProfileServiceImpl.java:27]: just for info, an alternative solution would be to use annotation on entity @SQLDelete

File: finance_manager/src/main/java/com/manager/finance/wallet/dao/WalletDao.java [hunk 4]
```diff
@@ -5,14 +5,16 @@
 import com.manager.finance.wallet.model.entity.api.DefaultCurrency;
 import java.util.List;
 import java.util.Optional;
+import org.springframework.data.domain.Sort;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface WalletDao extends JpaRepository<Wallet, Long> {
-    List<Wallet> findWalletByProfileId(Profile profileId);
+    List<Wallet> findWalletByProfileIdAndIsDeleted(Profile profileId, Boolean isDeleted, Sort sort);
```
Comment (AlexPravda1) [inline finance_manager/src/main/java/com/manager/finance/wallet/dao/WalletDao.java:12]: if, in general, we want to select only active wallets, I would suggest using @Where  annotation on entity

@Where(clause = "is_deleted=false")

**Review by AlexPravda1 — APPROVED** [review]

File: finance_manager/src/main/resources/db/changelog/changeset/v.0.0.2/update-user-login.sql [hunk 5]
```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+--changeset "gs":3
+
+UPDATE public.users SET u_email='sergey@exadel.com' WHERE u_email='Sergey@exadel.com';
+UPDATE public.users SET u_email='katia@exadel.com' WHERE u_email='Katia@exadel.com';
+UPDATE public.users SET u_email='jura@exadel.com' WHERE u_email='Jura@exadel.com';
+UPDATE public.users SET u_email='anna@exadel.com' WHERE u_email='Anna@exadel.com';
```
Comment (GapSerg) [inline finance_manager/src/main/resources/db/changelog/changeset/v.0.0.2/update-user-login.sql:7]: Why do you change the first letter of email?

**Review by GapSerg — APPROVED** [review]

**Pull request merged**